pkgs/development/tools/kafka-delta-ingest/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, perl
+, rdkafka
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kafka-delta-ingest";
+  version = "git-2021-12-08";
+
+  src = fetchFromGitHub {
+    owner = "delta-io";
+    repo = pname;
+    rev = "c48c854145b5aab1b8f36cc04978880794a2273c";
+    sha256 = "sha256-q0jOVZlxMHIhnc8y2N8o7Sl5Eg7DfJ96kXrPIV8RD1Y=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    perl
+  ];
+
+  buildFeatures = [ "dynamic-linking" ];
```

Comment:
Yes. Otherwise it tries to build its own librdkafka:

```
  --- stdout
  Cloning librdkafka
  Configuring librdkafka

  --- stderr
  Building and linking librdkafka statically
  Running command: "cp -a librdkafka/. /build/source/target/x86_64-unknown-linux-gnu/release/build/rdkafka-sys-0676a2bdf99e3b39/out" in dir: .
  Running command: "/build/source/target/x86_64-unknown-linux-gnu/release/build/rdkafka-sys-0676a2bdf99e3b39/out/configure --enable-ssl --disable-gssapi --disable-zstd --disable-lz4-ext" in dir: /build/source/target/x86_64-unknown-linux-gnu/release/build/rdkafka-sys-0676a2bdf99e3b39/out
  thread 'main' panicked at 'Command failed with error: No such file or directory (os error 2)', /build/kafka-delta-ingest-git-2021-12-08-vendor.tar.gz/rdkafka-sys/build.rs:36:19
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```